### PR TITLE
My First App section

### LINKED ISSUE #27: Add "My First App" section for the webinar

### PR DESCRIPTION
**Pull request by skymakerjaime**
Closes #27

### CODE REVIEW

**Comment by skymakerjaime** [comment]
Alex said it was awesome

File: docs/my-first-app.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+## 2. Know your component
+
+Understanding how the component is structured is the key to create your best app. For that:
+- Under the section _Components_, in the component _BoxTemplate_ click on _Edit_ to open.
+- Here we find two sections: the _Preview section_ (left side) and the _Code section_ (right side).
+- In the _Code section_, there are many tabs but we will focus on two: _Component_ and _Parameters_.
+- Under _Parameters_ you will find the 3 parameters that are being used for the UI and modify the model. 
+- Under _Components_ you will find how the component is created, its properties and more.
+
+[reference plane](reference-plane.md)
```
Comment (axelSkyMaker) [inline docs/my-first-app.md]: Why is this here?

File: docs/my-first-app.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  - Select the template you want to start from (now it will be _BoxTemplate_ by default).
+  - Select the _Access_ you want to give to your app (_Open_ is the only one available for the _Community_ account)
+
+![Import Component](assets/my-first-app/my-first-app.gif)
+
+By clicking in _Edit_ you can see the dashboard of your app, consisting of four different parts:
+-  _User Interfaces_: your final application.
+-  _Components_: your components that the application can include.
+-  _Drawings_: your drawings generated as pdf or dxf.
+- _Settings & Deploy_: on the right side including specific settings for your app and its final deploy to use it.
```
Comment (axelSkyMaker) [inline docs/my-first-app.md]: I'm having trouble reading this sentence. "and its final deploy to use it"?

File: docs/my-first-app.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+It is good to see that your changes are correct and the model behaves as expected. Then:
+
+ - Click on _Test Deploy_.
+ - If you _Published_ your component successfully, you will see the new model.
+ - After checking the parameters, you could realize about three things:
+
+      - The hole should be auto-centered, not always in the bottom-left corner.
+      - `Length` & `Width` should be min `50`  to have a rectangular hole and max `120` since you may not want a very large pen holder.
+      - `Width` parameter could be better as a [slider](configuration.md#slider) instead of an [input](configuration.md#input) parameter.
+      -
```
Comment (axelSkyMaker) [inline docs/my-first-app.md]: Remove
